packages/twenty-server/src/core/user-workspace/user-workspace.entity.ts

```diff
@@ -22,15 +23,30 @@ export class UserWorkspace {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
+  // @Field(() => [User])
+  // @ManyToMany(() => User, (user) => user.userWorkspaces)
+  // users: User[];
+
+  @Field(() => User)
   @JoinColumn({ name: 'userId' })
   user: User;
```

Comment:
I think we need a ManyToOne relation here to complete the ManyToMany between User and Workspace entities